docs/source/terminology.rst

```diff
@@ -38,7 +38,7 @@ Determining the salient elements for a gene fusion is dependent upon the context
 
 Assayed Gene Fusions
 ####################
-Assayed gene fusions from biological specimens are directly detected using RNA-based gene fusion assays, or alternatively may be inferred from genomic rearrangements detected by whole genome sequencing or by coarser-scale cytogenomic assays in the context of informative phenotypic biomarkers. For example, an EWSR1 fusion is often inferred by breakapart FISH assay when a neoplasm is diagnosed or suspected to be Ewing sarcoma/primitive neuroectodermal tumor by immunohistochemical and/or morphological analysis.
+Assayed gene fusions from biological specimens are directly detected using RNA-based gene fusion assays, or alternatively may be inferred from genomic rearrangements detected by whole genome sequencing or cytogenomic assays in the context of informative phenotypic biomarkers. For example, an EWSR1 fusion is often inferred by breakapart FISH assay when a neoplasm is diagnosed or suspected to be Ewing sarcoma/primitive neuroectodermal tumor by immunohistochemical and/or morphological analysis.
```

Comment:
YA: I removed “by coarser scale” since it seems derogatory.  Also, in an effort to keep a global perspective on this, some countries have only these cytogenomics tools to detect fusions. If we prefer to comment on the resolution, we can add “low-resolution” cytogenomics assays.